lib/rules/function-url-scheme-blacklist/index.js

```diff
@@ -0,0 +1,76 @@
+"use strict"
+
+const containsString = require("../../utils/containsString")
+const functionArgumentsSearch = require("../../utils/functionArgumentsSearch")
+const isStandardSyntaxUrl = require("../../utils/isStandardSyntaxUrl")
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+const _ = require("lodash")
+const parse = require("url").parse
+
+const ruleName = "function-url-scheme-blacklist"
+
+const messages = ruleMessages(ruleName, {
+  rejected: scheme => `Unexpected url scheme "${scheme}:"`,
+})
+
+const rule = function (blacklist) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: blacklist,
+      possible: [_.isString],
```

Comment:
Readme says it accepts `array|string`. I'm not sure if it checks that. There're no tests if only one `string` was in config (not array with one `string`).